### DISCUSSION TITLE
Documentation-v2-Feedback

### DISCUSSION THREAD

**Discussion by nick-wireless** [harlan-zw/nuxt-seo#116] [General]
Hi Harlan,

I've picked up version 2 from version 1.  I think the updated direction and documentation is solid.

I wonder if at this location: [nuxt-seo-kit](https://github.com/harlan-zw/nuxt-seo-kit/tree/v2)/[docs](https://github.com/harlan-zw/nuxt-seo-kit/tree/v2/docs)/[content](https://github.com/harlan-zw/nuxt-seo-kit/tree/v2/docs/content)/[0.nuxt-seo](https://github.com/harlan-zw/nuxt-seo-kit/tree/v2/docs/content/0.nuxt-seo)/[1.getting-started](https://github.com/harlan-zw/nuxt-seo-kit/tree/v2/docs/content/0.nuxt-seo/1.getting-started/migration-guide.md) it makes more sense to display the code block of V2 first in the migration guide discussing nuxt.config settings.

Reading this it felt like V1 was the preferred implementation standard?  (I am also guessing that the simplification for url details is `url : process.env.NUXT_PUBLIC_SITE_URL`  which would be required?)  Not a bigy but gave me a moment of pause when everything else seemed to roll smoothly on the page.

**Comment by nick-wireless** [comment]
I wonder also if... showing the change of `language` , transitioning to `defaultLocale` is a good example to include in the code blocks?

**Comment by nick-wireless** [comment]
### Link - to Schema.org

- I wonder if [Schema full list](https://schema.org/docs/full.html) is the link to update to?
- I see there are many schemas beyond people and corporations... I wonder if it is an error to expand too much in the generic tags for the whole site and or for a particular page.
- Use case in my instance is to call out I am writing website for a 'nonprofit' organisation, they appear to link to a particular page re. organisations status in the examples.  I wonder if in this instance it is better to stay silent on the status for website as a whole and overwrite / extend for a particular page?

**Comment by harlan-zw** [comment]
Hey @nick-wireless 

Really appreciate the feedback! Thanks for taking the time to write it up. I'll make sure to either action or reply to all of these items before the v2 is released as a stable.
